taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from taxi.views import index
+
+urlpatterns = [
+    path("", index, name="index"),
+]
+
+app_name = "homepage"

```

Comment:
your `app_name` is `taxi`) `homepage` is not a good name as you will have URLs not only to home page